packages/light-client/src/spec/utils.ts

```diff
@@ -0,0 +1,47 @@
+import {BitArray, byteArrayEquals} from "@chainsafe/ssz";
+import {FINALIZED_ROOT_DEPTH, NEXT_SYNC_COMMITTEE_DEPTH} from "@lodestar/params";
+import {altair, phase0, ssz} from "@lodestar/types";
+
+export const GENESIS_SLOT = 0;
+export const ZERO_HASH = new Uint8Array(32);
+export const ZERO_PUBKEY = new Uint8Array(48);
+export const ZERO_SYNC_COMMITTEE = ssz.altair.SyncCommittee.defaultValue();
+export const ZERO_NEXT_SYNC_COMMITTEE_BRANCH = Array.from({length: NEXT_SYNC_COMMITTEE_DEPTH}, () => ZERO_HASH);
+export const ZERO_HEADER = ssz.phase0.BeaconBlockHeader.defaultValue();
+export const ZERO_FINALITY_BRANCH = Array.from({length: FINALIZED_ROOT_DEPTH}, () => ZERO_HASH);
+/** From https://notes.ethereum.org/@vbuterin/extended_light_client_protocol#Optimistic-head-determining-function */
+const SAFETY_THRESHOLD_FACTOR = 2;
+
+export function sumBits(bits: BitArray): number {
```

Comment:
This function is defined in 3 different places across the code base now. Here, in the beacon node package, and in the light client src. Might be good to have it only in the util package.

Reply:
Such a tinny function, which is just an alias for readibility on the SSZ method. I think it's better to have 2 extra lines than a random function in the utils package.